src/js/components/projects/detail.tsx

```diff
@@ -196,11 +197,18 @@ const ProjectDetail = (
                     </div>
                   </div>
                 ) : (
-                  <Button
-                    label={i18n.t('Create Scratch Org')}
-                    variant="outline-brand"
-                    onClick={openCreateOrgModal}
-                  />
+                  <div>
+                    <Button
+                      label={i18n.t('Create Scratch Org')}
+                      variant="outline-brand"
+                      onClick={openCreateOrgModal}
+                    />
+                    <TourPopover
+                      align={'bottom'}
+                      heading={'some heading'}
+                      body={'some body text'}
```

Comment:
@dvdherron When you're passing in a raw string, you don't need the `{}` brackets -- so `align="bottom"` would work here too.

However, any user-facing strings (so I think `heading` and `body`) should be run through the i18n setup, so I think we eventually want e.g.:

```suggestion
                      align="bottom"
                      heading={i18n.t('some heading')}
                      body={i18n.t('some body text')}
```